python/ray/autoscaler/node_provider.py

```diff
@@ -38,7 +40,12 @@ def load_aws_config():
 
 
 def get_node_provider(provider_config, cluster_name):
+    if provider_config["type"] == 'external':
+        provider_cls = importlib.import_module(name=provider_config["module"])
```

Comment:
Does this work for both Python 2 and 3?

Reply:
Yes, `importlib` is available for both Python2/3.